common/provider-ocp-add-container.adoc

```diff
@@ -62,7 +62,12 @@ In OpenShift, the default deployment of the router generates certificates during
 +
 [NOTE]
 ====
-These settings download the image inspector container image from the registry and obtain the Common Vulnerabilities and Exposures (CVE) information (for effective scanning) via a proxy.
+These settings download the image inspector container image from the registry and obtain the Common Vulnerabilities and Exposures (CVE) information (for effective scanning) via a proxy. This requires {product-title_short} appliance to have external access to reach the Internet; the URLs required by {product-title_short} to be open for OpenSCAP scanning are: 
```

Comment:
"these settings download ... and obtain" is grammatically weird.
"these settings control downloading ... and obtaining" perhaps?

Reply:
Fixed, thanks.